packages/admin/src/components/Navigation/index.tsx

```diff
@@ -1,10 +1,14 @@
-import { useContext } from "react";
-import { Scrapers, ScraperContext } from "@admin/utils/scraperContext";
+import { Scrapers } from "@admin/store/scraperEnum";
+import { useAppDispatch } from "@admin/store";
+import { setScraper, view } from "@admin/store/viewSlice";
 import { cx } from "@emotion/css";
+import { useAppSelector } from "../../store";
```

Comment:
혹시 이 부분도 `@admin/store`로 바꾸는 게 어떨까요?
```suggestion
import { useAppSelector } from "@admin/store";
```

Reply:
앗 반영하겠슴당